site/snippets/portal.ts

```diff
@@ -1,21 +1,14 @@
-import { WalletClientSigner, type SmartAccountSigner } from "@alchemy/aa-core";
-import Portal, { PortalOptions } from "@portal-hq/web";
-import { createWalletClient, custom } from "viem";
-import { polygonMumbai } from "viem/chains";
+import { PortalSigner } from "@alchemy/aa-signers";
+import { sepolia } from "viem/chains";
 
-const portalOptions = {
-  autoApprove: true,
-  gatewayConfig: `${polygonMumbai.rpcUrls.alchemy.http}/${process.env.ALCHEMY_API_KEY}`,
-  chainId: 80001,
-  host: process.env.PORTAL_WEB_HOST,
```

Comment:
is this not necessary anymore?

Reply:
nope it was optional on their constructor params, but fair can add it back if necessary